website/blog/2018-06-04-fxa-train-112.md

```diff
@@ -0,0 +1,96 @@
+Hi All,
+
+Last week we shipped FxA train-112 to production,
+with the following highlights:
+
+## FxA-0: quality
```

Comment:
@shane-tomlinson could you truncate the article before this first heading: https://github.com/mozilla/application-services/commit/bf068acd7c85fb072292b89e74b9c33d37c90d7e